yarn.lock

```diff
@@ -17132,6 +17118,20 @@ plop@^4.0.1:
     ora "^8.0.0"
     v8flags "^4.0.1"
 
+plop@^2.7.4:
```

Comment:
where does this come from? let's upgrade the lib of it too (icons I guess?)

Reply:
yeah of course, once we'll separate it to a different lib I meant, sorry
cool 💪🏼